internal/docstore/dynamodocstore/example_test.go

```diff
@@ -16,25 +16,47 @@ package dynamodocstore_test
 
 import (
 	"context"
-	"fmt"
+	"log"
 
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"gocloud.dev/internal/docstore"
 	"gocloud.dev/internal/docstore/dynamodocstore"
 )
 
 func ExampleOpenCollection() {
-	ctx := context.Background()
+	// This example is used in https://gocloud.dev/howto/docstore.
+
+	// import _ "gocloud.dev/docstore/dynamodocstore"
+
 	sess, err := session.NewSession()
 	if err != nil {
-		fmt.Println(err)
-		return
+		log.Fatal(err)
 	}
-	coll, err := dynamodocstore.OpenCollection(dynamodb.New(sess), "docstore-test", "_id", "", nil)
+	coll, err := dynamodocstore.OpenCollection(
+		dynamodb.New(sess), "docstore-test", "partitionKeyField", "", nil)
 	if err != nil {
-		fmt.Println(err)
-		return
+		log.Fatal(err)
 	}
-	errs := coll.Actions().Put(map[string]interface{}{"_id": "Alice", "score": 25}).Do(ctx)
-	fmt.Println(errs)
+
+	// Ignore unused variables in example:
+	_ = coll
```

Comment:
`coll.Close()`? If such a method doesn't exist now, it should (see #1492 and #1817), but we should have a separate issue about that.

Reply:
#2310